rfd/0170-nested-accesslists.md

```diff
@@ -147,24 +173,26 @@ spec:
     roles:
     - manager
     traits: {}
-  # list of references to other access lists, for users to include in this access list
-  dynamic_members:
-    access_list_members: "acl-a"
+  members:
+    - acl-a
   title: access-list-a
 version: v1
 
 ```
 
-When calculating a users access roles, the tree of access lists will
-be traversed and so upon login, user Alice will have been granted
+When calculating a user's access roles, the tree of Access Lists will
+be traversed, and so, upon login, user Alice will be granted:
 
 - `some-role` from `acl-a`
-- `manager` from `acl-c` as it includes members of `acl-a`
-- `auditor` and `reviewer` from `acl-b` as it includes members of
-  `acl-c`, who also include members of `acl-a`
+- `manager` from `acl-c`, as it includes members from `acl-a`
+- `auditor` and `reviewer` from `acl-b`, as it includes members from
+  `acl-c`, which in turn includes members from `acl-a`
 
 # Future considerations
 
-Further development of membership reviews should be considered to
+- Further development of membership reviews should be considered to
 expand reviews to include members of nested lists and to include
-information in the UI that nested may need to be separately reviewed
+information in the UI that nested lists may need to be separately reviewed.
+- Adding reverse-pointers to Access List resources for ancestor list 
```

Comment:
Haven't you just added this in https://github.com/gravitational/teleport/pull/47828? Should we remove this from "future considerations"?

Reply:
Yup